### ISSUE TITLE
is_secure=False can change url unwanted

### ISSUE THREAD

**Issue by domrim** [joeirimpan/zammad_py#141]
The attribute `is_secure` of `ZammadAPI()` can modify the `host`

If the `host` contains the string "https" it will be replaced by "http":
https://github.com/joeirimpan/zammad_py/blob/d2566b1a71c9245b856ab68f3efd3baaf8396461/zammad_py/api.py#L22-L24

i would propose to set an `protocol` attribute which is set to "http" or "https" depending on the value of `is_secure` and then include it into the url-string:
```python
if is_secure:
    protocol = "https"
else:
    protocol = "http"
self.url = '%s://%s/api/v1/' % (protocol, host)
```

**Comment by joeirimpan** [comment]
@domrim Correct. Might as well allow URL to be taken instead of host.

**Comment by joeirimpan** [comment]
this is fixed now. You can start using full url instead of hostname from now. Released [zammad-py 1.0.1](https://pypi.org/project/zammad-py/)
